core/misc/tabledrag.js

```diff
@@ -618,26 +618,47 @@ Backdrop.tableDrag.prototype.dropRow = function (event, self) {
   }
 };
 
-
 /**
  * Get the coordinates from the event (allowing for browser differences).
  */
 Backdrop.tableDrag.prototype.pointerCoords = function (event) {
+
+  // Match both null and undefined, but not zero, by using != null.
+  // See https://stackoverflow.com/questions/2647867/how-to-determine-if-variable-is-undefined-or-null
+  if (event.pageX != null && event.pageY != null) {
+    return {x: event.pageX, y: event.pageY};
+  }
+
   // Complete support for pointer events was only introduced to jQuery in
   // version 1.11.1; between versions 1.7 and 1.11.0 pointer events have the
-  // clientX and clientY properties undefined. In those cases, the properties
-  // must be retrieved from the event.originalEvent object instead.
-  var clientX = event.clientX || event.originalEvent.clientX;
-  var clientY = event.clientY || event.originalEvent.clientY;
+  // pageX and pageY properties undefined. In those cases, the properties must
+  // be retrieved from the event.originalEvent object instead.
+  if (event.originalEvent && event.originalEvent.pageX != null && event.originalEvent.pageY != null) {
+    return {x: event.originalEvent.pageX, y: event.originalEvent.pageY};
+  }
 
-  if (event.pageX || event.pageY) {
-    return { x: event.pageX, y: event.pageY };
+  // Some old browsers do not support MouseEvent.pageX and *.pageY at all.
+  // See https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/pageY
+  // For those, we look at event.clientX and event.clientY instead.
+  if (event.clientX == null || event.clientY == null) {
+    // In some jQuery versions, some events created by jQuery do not have
```

Comment:
Not sure if this applies to Backdrop, since we are using an up to date jQuery version.

@quicksketch ??

Reply:
Yeah I think we may not need this entire PR, as it only affects earlier versions of jQuery we were not using. We should read the original Drupal issue and see if we can reproduce the problem as described there in Backdrop. If not, I think we should skip this whole PR.